src/Repository/ContentRepository.php

```diff
@@ -81,10 +102,10 @@ public function findOneBySlug(string $slug): ?Content
 //        ->setParameter('values',['red','yellow']);
     }
 
-    private function createPaginator(Query $query, int $page, int $max): Pagerfanta
+    private function createPaginator(Query $query, int $page, ?int $amountPerPage): Pagerfanta
     {
         $paginator = new Pagerfanta(new DoctrineORMAdapter($query));
-        $paginator->setMaxPerPage($max);
+        $paginator->setMaxPerPage($amountPerPage ?: 6);
```

Comment:
magic number

Reply:
Yes, will need to come from config. Will add a `@todo` if it's not easy to do right now.

Reply:
if it's not easy to do right now please move it to const

Reply:
I've gotten rid of it :-)